ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -0,0 +1,113 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.checks import AgentCheck
+
+from six import iteritems
+
+try:
+    import pymqi
+except ImportError:
+    pymqi = None
+
+from . import errors, metrics
+
+
+class IbmMqCheck(AgentCheck):
+
+    METRIC_PREFIX = 'ibm_mq'
+
+    SERVICE_CHECK = 'ibm_mq.can_connect'
+
+    QUEUE_MANAGER_SERVICE_CHECK = 'ibm_mq.queue_manager'
+    QUEUE_SERVICE_CHECK = 'ibm_mq.queue.can_connect'
+
+    def check(self, instance):
+        if not pymqi:
+            self.log.error("You need to install pymqi")
+            raise errors.PymqiException("You need to install pymqi")
+
+        channel = instance.get('channel')
```

Comment:
Is this required e.g. fails if `None`?

Reply:
it'll fail you need a channel

Reply:
yes its required

Reply:
I can make the failure more friendly but it'll fail if you don't include it

Reply:
`from datadog_checks.base import ConfigurationError`